### ISSUE TITLE
Grid: Menus can't be used with a keyboard in grid, listbuilder (2.1.1, 2.4.3)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#936]
### Expected behavior
Menu items can be reached and activated using just a keyboard. This is true for all of the dropdown menus (sort, context menu, view switcher) used in list builder and grids.

### Actual behavior
The menu opens, but focus cannot go into the menu items using the keyboard. This just seems to be affecting the grid demo. The sort component itself does not exhibit this issue. Filing issue as I don't know if this issue would affect every grid using the sort component or if there is a way to mitigate implementing the sort and affecting keyboard use.

**Comment by Blackbaud-PatrickOFriel** [comment]
This is an issue because these dropdowns are appended to the body so we'll have to solve that across the board.
